lib/wallaroo/core/source/connector_source/connector_framed_source_notify.pony

```diff
@@ -287,8 +290,9 @@ class ConnectorSourceNotify[In: Any val]
       _send_acks(source)
     end
 
+    let data': Array[U8] val = consume data
+    let data'': Array[U8] val = recover data'.clone() end
```

Comment:
Does the compiler allow this second var to be wrapped with `ifdef debug`?

Reply:
No.

Reply:
No, but there's a workaround:
have the "debug array" as a class variable, and _only populate it_ inside an `ifdef debug`. That way the print at the bottom always has the var, and it's mostly empty otherwise?

Reply:
I think from a PR perspective, we probably shouldn't have this print in there at all